src/dialogs/dialog-select-action.ts

```diff
@@ -181,8 +194,11 @@ export class DialogSelectAction extends LitElement {
         </mwc-list-item>
         `)}
       `;
-    }
-    else {
+  }
+
+  _renderDomainActions() {
+      if (!this._params?.domainFilter) throw new Error("Params with domainFilter needs to be set");
```

Comment:
we cannot throw an error here. there is no corresponding catch section.
it would be better to render an empty results list for this case.